src/scripting/flash/system/flashsystem.cpp

```diff
@@ -176,6 +197,117 @@ ASFUNCTIONBODY_ATOM(Capabilities,_getServerString)
 	*/
 	ret = asAtomHandler::fromString(sys,res);
 }
+
+ASFUNCTIONBODY_ATOM(Capabilities,_hasAudio)
+{
+    LOG(LOG_NOT_IMPLEMENTED,"Capabilities.hasAudio always returns true");
+    ret = asAtomHandler::fromBool(true);
+}
+
+ASFUNCTIONBODY_ATOM(Capabilities,_hasAudioEncoder)
+{
+    LOG(LOG_NOT_IMPLEMENTED,"Capabilities.hasAudioEncoder always returns true");
+    ret = asAtomHandler::fromBool(true);
+}
+
+ASFUNCTIONBODY_ATOM(Capabilities,_hasEmbeddedVideo)
+{
+    LOG(LOG_NOT_IMPLEMENTED,"Capabilities.hasEmbeddedVideo always returns true");
+    ret = asAtomHandler::fromBool(true);
+}
+
+ASFUNCTIONBODY_ATOM(Capabilities,_hasIME)
+{
+    LOG(LOG_NOT_IMPLEMENTED,"Capabilities.hasIME always returns true");
+    ret = asAtomHandler::fromBool(true);
+}
+
+ASFUNCTIONBODY_ATOM(Capabilities,_hasMP3)
+{
+    LOG(LOG_NOT_IMPLEMENTED,"Capabilities.hasMP3 always returns true");
+    ret = asAtomHandler::fromBool(true);
```

Comment:
We should return true only if we are compiling with ffmpeg enabled (#ifdef ENABLE_LIBAVCODEC)